packages/core/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    presets: ['module:metro-react-native-babel-preset'],
```

Comment:
yes, it is needed, otherwise `jest` is unable to run test (there is error at the babel parse step). Probably it is possible to solve it by writing rules for the babel, but it will be more configuration, so applying `metro-react-native-babel-preset` is the easiest (even though metro is not used for the packages).